setup.py

```diff
@@ -19,18 +19,18 @@
 Python, but it contains a full API for sending events from any language, in
 any application.
 
-:copyright: (c) 2011-2014 by the Sentry Team, see AUTHORS for more details.
-:license: BSD, see LICENSE for more details.
+:copyright: (c) 2011-2019 by the Sentry Team, see AUTHORS for more details.
+:license: BSL-1.1, see LICENSE for more details.
```

Comment:
Can you just delete this part tbh, we removed this copyright / license header everywhere else

Reply:
I felt like the setup.py was a special case, but yeah, I can totally remove this entire docstring altogether. We already have the license in setup args.

Reply:
We can remove this as it is covered below and in the repo directly.